src/sound/ScratchSoundPlayer.as

```diff
@@ -160,7 +170,9 @@ public class ScratchSoundPlayer {
 		for (i = 0; i < 4096; i++) {
 			var n:Number = interpolatedSample();
 			data.writeFloat(n);
+			dataBytes.writeFloat(n);
 			data.writeFloat(n);
+			dataBytes.writeFloat(n);
 		}
```

Comment:
I think it would be more efficient to say `dataBytes.writeBytes(data);` after the loop -- but only if `evt.data` starts empty, which I'm not sure of.